docs/upgrade/upgrade_to_v0.18.0.md

```diff
@@ -0,0 +1,97 @@
+# Upgrade to v0.18.0
+
+There are breaking changes between OpenPAI v0.14.0 and v0.18.0. Before v0.18.0, OpenPAI was based on Yarn and Kubernetes, and data was managed by HDFS. In v0.18.0, it has switched to a pure Kubernetes-based architecture. Many new features, such as `AAD authorization`, `Hivedscheduler`, `Kube Runtime`, `Marketplace`, etc., are also included in this release.
+
+The upgrade contains three phases:
+
+  1. Save previous data to a different place.
+  2. Remove previous PAI deployment.
+  3. Deploy PAI v0.18.0
+
+
+## Save your Data to a Different Place
+
+The upgrade from older version to `v0.18.0` cannot preserve any useful data: all jobs, user information, dataset will be lost inevitably and irreversibly. Thus, if you have any useful data in previous deployment, please make sure you have saved them to a different place.
```

Comment:
Suggest change:
The upgrade requires a manual backup of data, please ignore this step if you don't need to keep the data in original version. If not, please make sure you have saved the data for jobs and user information, dataset used by jobs to a temporary storage, and get them back later. The following section will guide you through how to back up the important data.

Reply:
The same as above. Users cannot "get them back later".